iron-overlay-behavior.html

```diff
@@ -167,8 +167,26 @@
       return this._focusedChild || Polymer.dom(this).querySelector('[autofocus]') || this;
     },
 
+    /**
+     * The deepest active element.
+     * @type Node
+     */
+    get _deepestActiveElement() {
+      var active = document.activeElement;
+      // In IE9,10,11 document.activeElement would return a non-HTMLElement when
+      // the overlay is inside an iframe, hence we should check for that.
+      if (active instanceof HTMLElement) {
```

Comment:
Without this check, tests fail with this error:
```
Windows 8.1 IE 11        ✖ test/iron-overlay-behavior.html » basic overlay tests » close an overlay by clicking outside
  Object doesn't support property or method 'focus' (data:text/javascript;charset=utf-8,%0A%0A%2F**%0AUse%20%60Polymer.IronOverlayBehavior%60%20to%20implement%20an%20element%20that%20can%20be%20hidden%20or%20shown%2C%20and%20displays%0Aon%20top%20of%20other%20content.%20It%20includes%20an%20optional%20backdrop%2C%20and%20can%20be%20used%20to%20implement%20a%20variety%0Aof%20UI%20controls%20including%20dialogs%20and%20drop%20downs.%20Multiple%20
// [...]
```
So basically `overlay._deepestActiveElement` is called after `attached` for that particular test, and `document.activeElement` is not an HTMLElement on IE11...

@notwaldorf @cdata any thoughts? I don't really like to put this check here just because of the tests :x